articles/defender-for-cloud/devops-faq.md

```diff
@@ -108,6 +109,11 @@ The following languages are supported by Defender for DevOps:
 - Java Script
 - Type Script
 
+### I'm getting an error in Azure DevOps saying there's no CLI tool
+If when running the pipeline in Azure DevOps, you receive the following error: 
+"no such file or directory, scandir 'D:\a\_msdo\versions\microsoft.security.devops.cli'".
+
+This error is because you are missing the dependency of dotnet6 in the pipeline's yaml file. DotNet6 is a requirement in order for the Microsoft Security DevOps extension to run, so you should include this as a task in your yaml file. See more [here](https://marketplace.visualstudio.com/items?itemName=ms-securitydevops.microsoft-security-devops-azdevops). 
```

Comment:
```suggestion
This error occurs if you are missing the dependency of `dotnet6` in the pipeline's YAML file. DotNet6 is required to allow the Microsoft Security DevOps extension to run. Include this as a task in your YAML file to eliminate the error.
 
You can learn more about [Microsoft Security DevOps](https://marketplace.visualstudio.com/items?itemName=ms-securitydevops.microsoft-security-devops-azdevops). 
```

Reply:
@LianaT these corrections are there to fit to Microsoft standards, please commit